src/util/errors-ts.ts

```diff
@@ -1056,7 +1056,7 @@ export class LambdaSizeExceededError extends NowError<
         size
       ).toLowerCase()}) exceeds the configured limit (${bytes(
```

Comment:
Maybe this should say `the maximum size` instead of `configured`.

Reply:
@AndyBitz Ahhh, good point. updating it.